cpp/src/aztec/plonk/proof_system/widgets/transition_widgets/transition_widget.hpp

```diff
@@ -206,8 +203,7 @@ class EvaluationGetter : public BaseGetter<Field, Transcript, Settings, num_widg
 
 /**
  * @brief Provides access to polynomials (monomial or coset FFT) for use in widgets
- * @details Coset FFT access is needed in quotient construction and monomial access is
- * for construction of linearization polynomial
+ * @details Coset FFT access is needed in quotient construction.
```

Comment:
Another shitty thing about GitHub: you cant comment on lines that don't appear in the diff... Anyway, in the lines below I think there's more logic that you can remove/simplify if you want. I think we do not need the `MONOMIAL == representation` case anymore which means `PolynomialGetter` can be simplified in a number of ways and we don't need to instantiate `MonomialGetter` later on. Yeesh. This is not a chill part of the code base.